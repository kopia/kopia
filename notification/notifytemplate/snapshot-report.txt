Subject: {{.EventArgs.OverallStatus}} on {{.Hostname}}

{{ range .EventArgs.Snapshots | sortSnapshotManifestsByName}}Path: {{ .Manifest.Source.Path }}

  Status:      {{ .StatusCode }}
  Start:       {{ .StartTimestamp | formatTime  }}
  Duration:    {{ .Duration }}
  Size:        {{ .TotalSize | bytes }}{{ .TotalSizeDelta | bytesDelta }}
  Files:       {{ .TotalFiles | formatCount }}{{ .TotalFilesDelta | countDelta }}
  Directories: {{ .TotalDirs | formatCount }}{{ .TotalDirsDelta | countDelta }}
{{ if .Error }}  Error:       {{ .Error }}
{{ end }}{{ if .Manifest.RootEntry }}{{ if .Manifest.RootEntry.DirSummary }}{{ if .Manifest.RootEntry.DirSummary.FailedEntries }}
  Failed Entries:
{{ range .Manifest.RootEntry.DirSummary.FailedEntries }}
  - {{.EntryPath}}: {{.Error}}{{ end }}{{ end }}{{ end }}
{{ end }}
{{ end }}Generated at {{ .EventTime | formatTime }} by Kopia {{ .KopiaBuildVersion }}.

https://kopia.io/