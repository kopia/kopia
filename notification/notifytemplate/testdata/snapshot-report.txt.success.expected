Subject: Successfully created a snapshot of /some/path on some-host

Path: /some/path

  Status:      success
  Start:       Thu, 02 Jan 2020 03:04:05 +0000
  Duration:    1.1s
  Size:        456 B (+56 B)
  Files:       123 (+23)
  Directories: 33 (+3)

  Failed Entries:

  - /some/path: some error
  - /some/path2: some error

Generated at Thu, 02 Jan 2020 03:04:05 +0000 by Kopia v0-unofficial.

https://kopia.io/